### PR TITLE
Add Okta game section on home page

### PR DESCRIPTION
**Pull request by volodymyrmakarchuk-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-345278](https://oktainc.atlassian.net/browse/OKTA-345278)

### Netlify deploy:
https://609284f85f5ed000bf3ea281--reverent-murdock-829d24.netlify.app/

### CODE REVIEW

**Comment by shawyu-okta** [comment]
It seems like the banner is too wide. Based on the mocks it seems like it should not extend over the navigation

**Comment by volodymyrmakarchuk-okta** [comment]
> It seems like the banner is too wide. Based on the mocks it seems like it should not extend over the navigation

I will remove this banner, banner part will be moved into another task

File: packages/@okta/vuepress-theme-prose/assets/css/components/_homePage.scss [hunk 1]
```diff
@@ -239,6 +241,55 @@
...
+  .homepage--okta-game-description {
+    flex-basis: 100%;
+    padding: 0 35px 0 0;
+    h3 {
+      color: #fff;
+      font-weight: normal;
+    }
+    p {
+      font-size: 1.5em;
+      line-height: 1.1em;
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/assets/css/components/_homePage.scss:260]: The spacing is a little bit tight on this area and I think the mock shows a slightly smaller font here.

Let's try 
```suggestion
      font-size: 1.3em;
      line-height: 1.3em;
```

**Review by shawyu-okta — COMMENTED** [review]
Two additional things: 
* `code_tycoon.png` can be removed for now since it's unused in this change
* The "Play Now" button text is not vertically centered - changing line height may help

**Review by shawyu-okta — APPROVED** [review]

**Pull request merged**